src/Frameworks/AsyncTableTools/AsyncTableTools/hooks/useTableSort/useTableSort.js

```diff
@@ -56,7 +56,7 @@ const useTableSort = (columns, options = {}) => {
   const [sortBy, setSortBy] = useTableState(
     TABLE_STATE_NAMESPACE,
     initialSortBy || {
-      index: 0,
+      index: 1,
```

Comment:
I thin this can be discarded. I think I typed `1`, because I forgot for a second indexes start with 0...

Reply:
I'll slap this up too